app/views/layouts/bk

```diff
@@ -0,0 +1,47 @@
+
+<body>
+
+  <%= render 'layouts/navigation' %>
```

Comment:
if this file is not used anywhere

Please remove it

Reply:
its is just a text file . I copied code before making changes so ...Ok I'll remove it